ext/bg/js/dictionary.js

```diff
@@ -177,11 +177,17 @@ function dictTermsMergeBySequence(definitions, mainDictionary) {
 
 function dictTermsMergeByGloss(result, definitions, appendTo=null, mergedIndices=null) {
     const definitionsByGloss = appendTo !== null ? appendTo : new Map();
+
+    const resultExpressionsMap = result.expressions;
```

Comment:
Can't comment on the exact line, but I think the data layout comment could also be updated to `resultExpressionsMap` instead of `result.expressions`